### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by Salma-Elmahallawy**
Description, Team Members, and techniques added to README.md

### CODE REVIEW

**Review by liborioc1 — APPROVED** [review]

**Review by liborioc1 — APPROVED** [review]

**Review by NabilahRouf — CHANGES_REQUESTED** [review]
To Do: Add Tarek's GitHub username

**Comment by melsagh** [comment]
Tarek needs to add his username

File: README.md [hunk 1]
```diff
@@ -1 +1,28 @@
...
+- Backend : Node.js and Express
+- Database : Firebase (NoSQL)
+
+## Contributors
+
+- Liborio Ciccarello (liborioc_1) 40095519 
+- Muhammad Al Sagh (melsagh) 40081836
+- Nabilah Rouf (NabilahRouf) 40095893
+- Salma Elmahallawy (Salma-Elmahallawy) 40091224
+- Tarek El Dick 40110697
```
Comment (liborioc1) [inline README.md:28]: Tarek' ID is missing

**Review by liborioc1 — CHANGES_REQUESTED** [review]
I think Objective and Description can be a bit more descriptive.

File: README.md [hunk 2]
```diff
@@ -1 +1,28 @@
...
+The core features are:
+1. Posting a picture (with text description)
+2. Following a user 
+3. Leave comments to an uploaded picture
+
+## Languages and Technologies
+
+- Frontend: Javascript (React)
+- Backend : Node.js and Express
+- Database : Firebase (NoSQL)
```
Comment (melsagh) [inline README.md:20]: we might go with Mongo though, right?

Comment (Salma-Elmahallawy) [reply README.md:20]: Yes, we can edit if we end up choosing it.

**Review by liborioc1 — CHANGES_REQUESTED** [review]
Tarik's username still missing

File: README.md [hunk 3]
```diff
@@ -1 +1,28 @@
-# Instagram_SOEN341
\ No newline at end of file
+# SOEN 341: Software Process
```
Comment (liborioc1) [inline README.md:1]: Worked on Issue #1  however, missing Username for Tarek

File: README.md [hunk 4]
```diff
@@ -4,15 +4,22 @@
 ## Objective
 
 The objective of this project is to become familiar with software development process through developing a web application.
+In this particular case, Agile Development (4 Sprints) will be the software development methodology used to implement the web application.
```
Comment (liborioc1) [inline README.md:8]: Worked on #1  Added more to objective and description however, github username is still missing for Tarek

Comment (TarekElDick) [reply README.md:8]: tarek needs to add his username

**Comment by TarekElDick** [comment]
tarek needs to add his username

**Comment by TarekElDick** [comment]
Will add my username

**Review by TarekElDick — APPROVED** [review]
will do !

**Review by TarekElDick — APPROVED** [review]

**Review by melsagh — APPROVED** [review]

**Review by liborioc1 — APPROVED** [review]

**Review by NabilahRouf — APPROVED** [review]

**Pull request merged**